articles/resources/research.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Q# in Research
+description: Learn about how to use Q# in research.
+author: cgranade
+ms.author: chgranad
+ms.date: 8/5/2020
+ms.topic: article
+uid: microsoft.quantum.research
+no-loc: ['Q#', '$$v']
+---
+
+# Q# in Research
+
+Q# and the Quantum Development Kit are important tools for empowering quantum computing research.
+On this page, we list a few resources to help you get the most out of Q# in your research.
```

Comment:
I'd tone down the introductory paragraph and make it a be more descriptive. This page is targeted to researchers and they tend to like a more aseptic language.